src/api/app/views/webui/request/_action_text.html.haml

```diff
@@ -39,3 +39,5 @@
         by
         = details['user']
         = render TimeComponent.new(time: details['time'].to_i)
+
+  %i.fas{ class: build_status_icon_from_action(action) }
```

Comment:
I think there is a missing header column for this, it should be added [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/request/_actions_details.html.haml#L19)